lemmy-ui/services.yml

```diff
@@ -0,0 +1,30 @@
+version: "3.7"
+
+x-logging: &default-logging
+  driver: "json-file"
+  options:
+    max-size: "50m"
+    max-file: "4"
+
+services:
+  ui:
+    image: dessalines/lemmy-ui:0.19.3
+    environment:
+      #- LEMMY_UI_DEBUG=true
+      - LEMMY_UI_HOST=0.0.0.0:3000
```

Comment:
NOTE: I changed from the default 1234 to 3000. This was to match the sublinks-ui and so I could reference either container as "ui" for purposes of the nginx config files. Only one of the 2 containers can be run (but not sure why you would ever need to run both?)